tsuru/client/deploy.go

```diff
@@ -645,3 +646,72 @@ func (c *AppDeployRebuild) Run(context *cmd.Context, client *cmd.Client) error {
 	}
 	return nil
 }
+
+type AppDeployRollbackUpdate struct {
+	cmd.GuessingCommand
+	image   string
+	reason  string
+	disable bool
+	fs      *gnuflag.FlagSet
+}
+
+func (c *AppDeployRollbackUpdate) Info() *cmd.Info {
+	desc := `Locks an existing image of an app. You can list images with "tsuru app-deploy-list -a <appName>"
+
+::
+
+    The [-i/--image] flag is the name of an app image.
+
+    The [-d/--disable] flag enables or disables a rollback to an image, by default if not parsed it enables a rollback to an image, otherwise it'll disable it.
+
+    The [-r/--reason] flag lets the user tell why this action was needed.
+`
+	return &cmd.Info{
+		Name:    "app-deploy-rollback-update",
+		Usage:   "app-deploy-rollback-update [-a/--app appName] [-i/--image imageName] [-d/--disable] [-r/--reason reason]",
+		Desc:    desc,
+		MinArgs: 0,
+		MaxArgs: 0,
+	}
+}
+
+func (c *AppDeployRollbackUpdate) Flags() *gnuflag.FlagSet {
+	if c.fs == nil {
+		c.fs = c.GuessingCommand.Flags()
+		image := "The image name of a version of a app"
+		c.fs.StringVar(&c.image, "image", "", image)
+		c.fs.StringVar(&c.image, "i", "", image)
+		reason := "A message describing this rollback"
+		c.fs.StringVar(&c.reason, "reason", "", reason)
+		c.fs.StringVar(&c.reason, "r", "", reason)
+		disable := "Enables or disables the rollback on a specific image version"
+		c.fs.BoolVar(&c.disable, "disable", false, disable)
+		c.fs.BoolVar(&c.disable, "d", false, disable)
+	}
+	return c.fs
+}
+
+func (c *AppDeployRollbackUpdate) Run(context *cmd.Context, client *cmd.Client) error {
+	appName, err := c.Guess()
+	if err != nil {
+		return err
+	}
+	u, err := cmd.GetURL(fmt.Sprintf("/apps/%s/deploy/rollback/update", appName))
+	if err != nil {
+		return err
+	}
+	v := url.Values{}
+	v.Set("image", c.image)
+	v.Set("reason", c.reason)
+	v.Set("origin", "rollback")
+	// inverted, cause when `disable == true`, in API `CanRollback == false `
```

Comment:
made some changes here (https://github.com/tsuru/tsuru/pull/1561/files#diff-97ed4f786bfbc6c5cf3bafb5d88368deR444) and here (https://github.com/tsuru/tsuru/pull/1561/files#diff-97ed4f786bfbc6c5cf3bafb5d88368deR436), so this will be direct without the need of inverting this variable.